src/test/java/bio/terra/FileTypeDetectorTest.java

```diff
@@ -34,7 +34,7 @@ public void tarGzipHandlerTest() {
   public void gzipHandlerTest() {
```

Comment:
This `gzipHandlerTest` and the `jsonHandlerTest` tests are both invalid. They set the input stream, then read the data stream, without ever calling `handleGzipType` or `handleJsonType`. Therefore, when they go to read the data stream, the data stream is `null`. But, the `catch (Exception e) {}` silently catches and ignores any errors.

Please change the catch clause to something like:
```
    } catch (Exception e) {
      fail("test failed with error: " + e.getMessage());
    }
```

Reply:
Good catch! Added back `handleMediaType()` to unit tests `gzipHandlerTest` and `jsonHandlerTest`.